pkg/discovery/endpoint/restapi/operations_test.go

```diff
@@ -151,10 +152,12 @@ func TestWebFinger(t *testing.T) {
 		var w restapi.WebFingerResponse
 
 		require.NoError(t, json.Unmarshal(rr.Body.Bytes(), &w))
-		require.Equal(t, "http://vct", w.Subject)
-		require.Equal(t, "http://vct", w.Links[0].Href)
-		require.Equal(t, "http://vct1", w.Links[1].Href)
-		require.Equal(t, "vct-v1", w.Properties["https://trustbloc.dev/ns/ledger-type"].(string))
```

Comment:
Where did the vct type get moved to?

Reply:
vct type will be in another webfinger document you can resolve the vct webfinger from here https://github.com/trustbloc/orb/compare/4571fae79534dd403e732f086002566886f3abcc..509025ceba26c0abcd00912b3047ceb33a1b3492#diff-d1651d44c95b6e4e251628b7eb01ccf823d6e88cbb0d37939ed531c0afbf1667R33